seqr/views/apis/staff_api_tests.py

```diff
@@ -260,6 +271,105 @@
+        }
+    ]}
+
+AIRTABLE_COLLABORATOR_RECORDS = {
+    "records": [
+        {
+            "id": "recW24C2CJW5lT64K",
+            "fields": {
+                "CollaboratorID": "Hildebrandt",
+            }
+        },
+        {
+            "id": "reca4hcBnbA2cnZf9",
+            "fields": {
+                "CollaboratorID": "Seidman",
+            }
+        }
+    ]
+}
+
+EXPECTED_PI_SUBJECT_FILE = [
+    u'1kg project n\xe5me with uni\xe7\xf8de_PI_Subject',
+    ['entity:subject_id', 'subject_id', 'prior_testing', 'project_id', 'pmid_id',
+     'dbgap_submission', 'dbgap_study_id', 'dbgap_subject_id', 'multiple_datasets', 'sex',
+     'ancestry', 'ancestry_detail', 'age_at_last_observation', 'phenotype_group', 'disease_id',
+     'disease_description', 'affected_status', 'onset_category', 'age_of_onset', 'hpo_present',
+     'hpo_absent', 'phenotype_description', 'solve_state']
+]
+EXPECTED_PI_SUBJECT_ROW = {
+    'project_guid': u'R0001_1kg', 'num_saved_variants': 0, 'dbgap_submission': 'No',
+    'hpo_absent': u'HP:0011675|HP:0001674|HP:0001508', 'solve_state': 'Unsolved', 'phenotype_group': '', 'sex': 'Male',
+    'phenotype_description': '', 'ancestry': '', 'ancestry_detail': '', 'entity:subject_id': u'NA19675_1',
+    'hpo_present': u'HP:0001631|HP:0002011|HP:0001636', 'multiple_datasets': 'No', 'onset_category': u'Adult onset',
+    'subject_id': u'NA19675_1', 'family_guid': u'F000001_1', 'affected_status': 'Affected', 'pmid_id': '',
+    'project_id': u'1kg project n\xe5me with uni\xe7\xf8de'}
+
+EXPECTED_PI_SAMPLE_FILE = [
+    u'1kg project n\xe5me with uni\xe7\xf8de_PI_Sample',
+    ['entity:sample_id', 'subject_id', 'sample_id', 'dbgap_sample_id', 'sample_source',
+     'sample_provider', 'data_type', 'date_data_generation']
+]
+EXPECTED_PI_SAMPLE_ROW = {'entity:sample_id': u'NA19675_1', 'data_type': u'WES', 'subject_id': u'NA19675_1', 'sample_provider': '', 'sample_id': u'NA19675', 'date_data_generation': '2017-02-05'}
+
+EXPECTED_PI_FAMILY_FILE = [
+    u'1kg project n\xe5me with uni\xe7\xf8de_PI_Family',
+    ['entity:family_id', 'subject_id', 'family_id', 'paternal_id', 'maternal_id', 'twin_id',
+     'family_relationship', 'consanguinity', 'consanguinity_detail', 'pedigree_image',
+     'pedigree_detail', 'family_history', 'family_onset']
+]
+EXPECTED_PI_FAMILY_ROW = {
+    'maternal_id': u'NA19679', 'subject_id': u'NA19675_1', 'consanguinity': 'Present',
+    'family_id': u'1', 'entity:family_id': u'NA19675_1', 'paternal_id': u'NA19678'}
+
+EXPECTED_PI_DISCOVERY_FILE = [
+    u'1kg project n\xe5me with uni\xe7\xf8de_PI_Discovery',
+    ['entity:discovery_id', 'subject_id', 'sample_id', 'Gene-1', 'Gene_Class-1',
+     'inheritance_description-1', 'Zygosity-1', 'Chrom-1', 'Pos-1', 'Ref-1', 'Alt-1', 'hgvsc-1',
+     'hgvsp-1', 'Transcript-1', 'sv_name-1', 'sv_type-1', 'significance-1']
+]
+EXPECTED_PI_DISCOVERY_ROW = {'entity:discovery_id': u'NA19675_1', 'subject_id': u'NA19675_1', 'sample_id': u'NA19675'}
```

Comment:
You need to test a discovery row that has gene data. In our call yesterday I specifically said this which is why I wanted to use saved variant 1. Its okay to instead use the other saved variant, but the here you should be using the row representing variant 2 to test against instead

Reply:
To have gene data in the discovery rows, I updated the genotype data of the saved_variant 2. Now it seems work. Please review.